.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Bug description
+Description of what the bug is.
+
+## Steps to reproduce
+Code or a description of how to reproduce the bug.
+
+## Environment
+- Operating system and version:
+- Python version:
```

Comment:
Installed Python package versions are one of the most common causes of issues we see.

```suggestion
- Python version:
- Output of `pip freeze`:
```